content/enterprise/install/automated/automating-initial-user.mdx

```diff
@@ -23,11 +23,12 @@ replicated admin --tty=0 retrieve-iact
 If you want to create the initial user in an automated deployment script, run a command like the following instead so that you can capture the IACT:
 
 ```shell
-initial_token=$(replicated admin retrieve-iact | tr -d '\r')
+initial_token=$(replicated admin --tty=0 retrieve-iact | tr -d '\r')
 ```
-
 The command outputs the complete IACT with the carriage return character removed, which facilitates use in automation.
 
+~> NOTE: The `--tty=0` flag enables the command to run successfully in automation, such as cloud-init. Without this flag set it will return an empty string.
```

Comment:
I don't know if this needs to be in a note. I think it could actually be more effective if we included it in the body of the text and had it come before the command to explain it, so users understand what it's doing and why. What if we did something like:

If you want to create the initial user in an automated deployment script, run a command that lets you capture the IACT. The following example command outputs the complete IACT with the carriage return character removed. The `--tty=0` flag enables the command to run successfully in automation, such as cloud-init. Without this flag set, the command will return an empty string.

command....

Reply:
Thanks @laurapacilio. I made the suggested change as looks great to me.